README.md

```diff
@@ -43,13 +43,13 @@ $ slather setup path/to/project.xcodeproj
 This will enable the `Generate Test Coverage` and `Instrument Program Flow` flags for your project.
 
 
-To test if you're ready to generate test coverage, run your test suite on your project, and then run:
+To verify if you're ready to generate test coverage, run your test suite on your project, and then run:
```

Comment:
When `test` is changed to `verify`, `if` no longer fits here.

I'd suggest removing that too:

```diff
- To verify if you're ready...
+ To verify you're ready...
```